### DISCUSSION TITLE
One question about the seed in the generated .tess file from Neper

### DISCUSSION THREAD

**Discussion by liuyii7** [neperfepx/neper#939] [Q&A]
Hi, I want to consult one question about the seed in the generated .tess file from Neper: 

I know that with the relative larger weight factor, the corresponding grain will have a larger size, but I want to consult the numerical relationship between the weight factor and the volume for each grain.

(Generally for the *seed part, the first column is the grain index, 2nd to 4th columns are the coordinates (3D case), and the 5th column is the weight factor)

**Comment by rquey** [comment]
It is the weight of the Laguerre tessellation, see [this paper](https://neper.info/papers.html#id2).

**Comment by liuyii7** [comment]
Thanks for your kindly reply, but it is not what I want to consult.

Actually I already seen the weight part in this paper: the weight represent the square of the radius of the sphere, not the generated grain. But I want to consult the specific relationship between the volume of grain and the weight of the seed. Actually I calculate the volume of the sphere based on its weight, and the result is far away with the volume of the corresponding generated grain.

**Reply by rquey** [reply]
There is no direct correlation between the two.

**Comment by liuyii7** [comment]
Okay I see, thanks!

Just one more thing: the reason why there is no direct correlation between these two factors is because of the optimization process (to avoid overlaps or gaps), or because of other reasons?

**Reply by rquey** [reply]
There is no correlation between the seed coordinates and weights (as there would be in the case of dense sphere packing). They are treated as independent (optimization) parameters. So, clustered seeds with large weights could generate smaller cells than scattered seeds with small weights, for example...

**Comment by liuyii7** [comment]
Okay I see, thank you so much!
